eofConvention.fsx

```diff
@@ -10,14 +10,22 @@ let EolAtEof(fileInfo: FileInfo) =
     else
         true
 
-let NotInDir (dirName: string) (fileInfo: FileInfo) =
-    not (fileInfo.FullName.Contains $"%c{Path.DirectorySeparatorChar}%s{dirName}%c{Path.DirectorySeparatorChar}")
+let InDir (dirName: string) (fileInfo: FileInfo) = 
+    fileInfo.FullName.Contains $"%c{Path.DirectorySeparatorChar}%s{dirName}%c{Path.DirectorySeparatorChar}"
+
+let NotInDirs (dirNames: List<string>) (fileInfo: FileInfo) =
+    not (dirNames 
+        |> Seq.map (fun dirName -> InDir dirName fileInfo)
+        |> Seq.contains true)
+
+let whitelistExtensions = [".svg"; ".png"; ".slnf"; ".so"; ".a"; ".dll"; ".pdb"; ".dylib"]
```

Comment:
@realmarv I said I want you to ignore all binary files, not create a binary file blacklist

Reply:
Done. 
Please note that in the "[FileConventions: implement HasBinaryContent](https://github.com/nblockchain/conventions/pull/34/commits/96b1d6a5e526de00163ee551d707442dd366270d)" commit the CI is red because eofConvention.fsx complains about binary files in DummyFiles (I used HasBinaryContent in the eofConvention.fsx later). if you click on the ci icon, the CI related to dotnet tests is green.